Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -1,5 +1,6 @@
 using System.Collections.Generic;
 using System.CommandLine;
+using System.Diagnostics.Tracing;
```

Comment:
Is the code using this?

Reply:
Nope ^,^ would prefer not to have another review/build cycle again for this though. Maybe we can find a better way of catching this in the future, maybe through `dotnet-format`.